druid/src/widget/split.rs

```diff
@@ -92,33 +102,33 @@ impl<T> Split<T> {
             }
         }
     }
+    fn calculate_limits(&self, size: Size) -> (f64, f64) {
+        let size_in_split_direction = match self.split_direction {
```

Comment:
Just fyi I believe you can do, `self.split_direction.major(size)` to get the relevant 'main axis length' for this direction.

Reply:
Nice that is cleaner! FYI: I had to make the function public (for the crate) though.

Reply:
I just noticed I actually had them the wrong way around as well...

Reply:
For anyone wondering what the answer is, as seen in a [previous discussion](https://xi.zulipchat.com/#narrow/stream/147926-druid/topic/split.20vertical.2Fhorizontal.20confusion):

The axis tells you the axis of the splitter itself.